### PR TITLE
Add PCC52: Pomodoro Timer realnitinworks

### PR DESCRIPTION
**Pull request by realnitinworks**
Difficulty level (1-10): [3]
Estimated time spent (hours): [3]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [I learned about the argparse module basics.]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 52/realnitinworks/pomodoro.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+pomodoro.py
+"""
+
+import sys
+import argparse
+from time import sleep
+from datetime import datetime, date, timedelta
+
+
+BREAK_URL = "https://www.youtube.com/watch?v=fDvDC5yNn6M"
```
Comment (bbelderbos) [inline 52/realnitinworks/pomodoro.py]: lol

File: 52/realnitinworks/pomodoro.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+DEFAULT_POMODORO_DURATION = 25 # minutes 
+DEFAULT_BREAK_DURATION = 2 # minutes 
+DEFAULT_POMODORO_COUNT = 5
+
+
+def log(msg):
+    timestamp = datetime.today()
+    print(f"[{timestamp}] {msg}")
```
Comment (bbelderbos) [inline 52/realnitinworks/pomodoro.py]: f-strings, nice

File: 52/realnitinworks/pomodoro.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+def video_break():
+    import webbrowser
```
Comment (bbelderbos) [inline 52/realnitinworks/pomodoro.py]: imports at the top?

File: 52/realnitinworks/pomodoro.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        help="number of pomodoros.",
+        type=int,
+        default=DEFAULT_POMODORO_COUNT
+    )
+    ap.add_argument(
+        "-b",
+        "--break-time",
+        help="duration of break time in minutes.",
+        type=int,
+        default=DEFAULT_BREAK_DURATION
```
Comment (bbelderbos) [inline 52/realnitinworks/pomodoro.py]: nice use of constants

File: 52/realnitinworks/pomodoro.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    )
+    args = vars(ap.parse_args())
```
Comment (bbelderbos) [inline 52/realnitinworks/pomodoro.py]: do we need `vars`?

**Comment by bbelderbos** [comment]
nice work @realnitinworks

File: 52/realnitinworks/pomodoro.py (hunk 3 above)
Comment (realnitinworks) [reply 52/realnitinworks/pomodoro.py]: ok. will do that from now on.

**Pull request merged**